WorkloadManagementSystem/Agent/test/Test_Agent_SiteDirector.py

```diff
@@ -172,6 +172,14 @@ def test__submitPilotsToQueue(mocker):
                                                 'Setup': 'LHCb-Production',
                                                 'Site': 'LCG.CERN.cern',
                                                 'SubmitPool': ''}}}
+
+  # Create a MagicMock that does not have the workingDirectory
+  # attribute (https://cpython-.readthedocs.io/en/latest/library/unittest.mock.html#deleting-attributes)
```

Comment:
I guess there is a small typo here:
https://cpython-test-docs.readthedocs.io/en/latest/library/unittest.mock.html#deleting-attributes

Other than that, that looks fine to me

Reply:
thanks !